scripts/protcon_analyisis/protcon_analyisis.yml

```diff
@@ -0,0 +1,72 @@
+  data_spatial_unit:
+    label: path to spatial unit 
+    description: String refers to the path string where the .csv file containing information required for the calculation of the indicator is located. There must be an ID for each spatial unit, name (if any) and the area of each one of them. If the analysis is done in time, specify a date, for example if the spatial units or nodes are protected areas, specify the date of their declaration.
+    type: text/csv
+    example: "/scripts/protcon_analyisis/input/spatialunits_studyarea_data.csv"
+  column_spatial_unit:
+    label: spatial unit 
+    description: String that refers to the name of the column in the spatial file with the levels used to define the spatial units. In cases where there are multiple polygons per level, they will be grouped together, and the attributes of the first listed polygon will be taken into account.
+    type: text
+    example: "spatial_unit"
+  column_area:
+    label: path to study area geometry  
+    description: Text string representing the file path to the spatial file containing the spatial boundaries pertaining to the study area. Accepted file types include shapefile (.shp), raster (.tif/GeoTIFF) or  GeoPackage (.gpkg). This path specifies the complete path location of the file within the file system and is utilized for identifying and accessing the spatial boundaries contained within the file.
+    type: text
+    example: "area_spatial_unit"
+  column_date:
+    label: date information column 
+    description: String that refers to the name of the column in the data_spatial_unit containing the date or time information corresponding to the spatial unit levels.
+    type: text
+    example: "created_date"
+  time_interval:
+    label: time intervals
+    description: Represents the periods or intervals used for analysis. It specifies the length and units of time that will be considered for analyzing the data. For example, time intervals like 10Y (10 years), 5Y (5 years), 4M (4 months) can be used to define the time granularity for conducting the analysis.
+    type: text
+    example: "10Y"
+  time_start:
+    label: start time
+    description: Refers to the starting date of the time_interval. If this value is null or not provided, the minimum date from the column_date will be automatically taken as the starting date for the analysis
+    type: text
+    example: null
+  time_end:
+    label: end time 
+    description: Refers to the ending date of the time_interval. If this value is null or not provided, the minimum date from the column_date will be automatically taken as the starting date for the analysis
+    type: text
+    example: null
+outputs:
+  protcon_result:
+    label: protcon results
+    description: "Contains the results of the connectivity analysis between the spatial units within the study area. This table provides valuable information about the spatial relationships and connectivity patterns among different spatial entities"
```

Comment:
Is there enough room to give a brief overview of what the format of the data is, i.e. what are the columns in the CSV?

Reply:
there is a better description and example of the protcon results in the markdown file